### PR TITLE
AuthZ: Implementation, tests and docs

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

**Review by kenany — APPROVED** [review]

**Comment by pangea-andrest** [comment]
Rebased on main and change target to `beta`

File: packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authz/AuthZClient.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
+package cloud.pangeacyber.pangea.authz;
+
+import cloud.pangeacyber.pangea.BaseClient;
+import cloud.pangeacyber.pangea.Config;
+import cloud.pangeacyber.pangea.authz.requests.*;
+import cloud.pangeacyber.pangea.authz.responses.*;
+import cloud.pangeacyber.pangea.exceptions.PangeaAPIException;
+import cloud.pangeacyber.pangea.exceptions.PangeaException;
+
+public class AuthZClient extends BaseClient {
```
Comment (caseysoftware) [inline packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authz/AuthZClient.java:10]: Can we have intro text like the Python class has - https://github.com/pangeacyber/pangea-python/pull/275/files#diff-57d60510b1ba51adc4228e295d12deb41729ba601342735faa37caeaf9bbfc9dR150 ?

And it should note that this is in Beta.

Comment (kenany) [reply packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/authz/AuthZClient.java:15]: Done.

**Review by pangea-andrest — COMMENTED** [review]
LGTM

**Pull request merged**